geode-wan/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ReplicateRegionCommandDUnitTest.java

```diff
@@ -210,90 +216,404 @@ public void testUnsuccessfulCancelReplicateRegionCommandInvocation_WithPartition
+    final int entries = 15000;
+    final String regionName = getRegionName(usePartitionedRegion);
+
+    // Site A
+    VM locatorInA = vm0;
+
+    VM server1InA = vm1;
+    VM server2InA = vm2;
+    VM server3InA = vm3;
+    List<VM> serversInA = Arrays.asList(server1InA, server2InA, server3InA);
+    final String senderIdInA = "B";
+
+    // Site B
+    VM locatorInB = vm4;
+    VM server1InB = vm5;
+    VM server2InB = vm6;
+    VM server3InB = vm7;
+    List<VM> serversInB = Arrays.asList(server1InB, server2InB, server3InB);
     VM client = vm8;
-    String senderIdInA = "B";
-    String senderIdInB = "C";
 
-    Integer senderLocatorPort = create3WanSitesAndClient(true, vm0,
-        vm1, vm2, serversInA, serverInB, serverInC, client,
-        senderIdInA, senderIdInB);
+    // Create locators
+    Integer locatorBPort = locatorInB.invoke(() -> WANTestBase.createFirstLocatorWithDSId(2));
+    Integer locatorAPort = locatorInA.invoke(() -> {
+      Properties props = getDistributedSystemProperties();
+      props.setProperty(DISTRIBUTED_SYSTEM_ID, "" + 1);
+      props.setProperty(REMOTE_LOCATORS, "localhost[" + locatorBPort + "]");
+      LocatorLauncherStartupRule launcherStartupRule =
+          new LocatorLauncherStartupRule().withProperties(props);
+      launcherStartupRule.start();
+      return launcherStartupRule.getLauncher().getPort();
+    });
 
-    int replicateRegionBatchSize = 20;
-    int entries = 20000;
-    Set<Long> keySet = LongStream.range(0L, entries).boxed().collect(Collectors.toSet());
-    String regionName = getRegionName(true);
-    // Put entries
-    client.invoke(() -> WANTestBase.doClientPutsFrom(regionName, 0, entries));
+    // Create servers and regions
+    createServersAndRegions(locatorBPort, serversInB, usePartitionedRegion, regionName, null);
+    createServersAndRegions(locatorAPort, serversInA, usePartitionedRegion, regionName,
+        senderIdInA);
 
+    // Create client
+    client.invoke(() -> WANTestBase.createClientWithLocator(locatorAPort, "localhost",
+        regionName, ClientRegionShortcut.PROXY));
 
-    // Check that entries are put in the region
+    // Put entries & verify result
+    client.invoke(() -> WANTestBase.doClientPutsFrom(regionName, 0, entries));
     for (VM member : serversInA) {
       member.invoke(() -> WANTestBase.validateRegionSize(regionName, entries));
     }
 
-    // Create senders and receivers with replication as follows: "A" -> "B" -> "C"
-    createSenders(true, serversInA, serverInB,
-        senderIdInA, senderIdInB);
-    createReceivers(serverInB, serverInC);
+    // Create senders and receivers with replication as follows: "A" -> "B"
+    if (useParallel) {
+      createReceiverInVMs(server1InB, server2InB, server3InB);
+      createSenders(useParallel, serversInA, null, senderIdInA, null);
+    } else {
+      // Senders will connect to receiver in server1InB
+      server1InB.invoke(() -> createReceiverWithMaximumTimeBetweenPings(-1));
```

Comment:
I would use createReceiverInVMs because otherwise it seems that it is important in this case to set MaximumTimeBetweenPings to -1